test/specs/upload/conf/upload/schema.json

```diff
@@ -1,152 +1,299 @@
+                        }
+                    }
+                },
+                {
+                    "comment": "asset/reference",
+                    "name": "file2_uri",
+                    "default": null,
+                    "nullok": false,
+                    "type": {
+                        "typename": "text"
+                    },
+                    "annotations": {
+                        "comment": [
+                            "thumbnail",
+                            "download"
+                        ],
+                        "tag:isrd.isi.edu,2017:asset": {
+                            "byte_count_column": "file2_bytes",
+                            "url_pattern": "/hatrac/js/ermrestjs/{{{key}}}/{{{file2_MD5}}}",
+                            "md5": "file2_MD5",
+                            "filename_column": "file2_name"
+                        },
+                        "tag:isrd.isi.edu,2016:visible-columns" : {
+                            "*" : ["key", "file1_uri", "file2_uri"]
+                        }
+                    }
+                }
+            ]
         },
-        {
-          "comment": null,
-          "name": "bytes",
-          "default": null,
-          "nullok": true,
-          "type": {
-            "typename": "int8"
-          },
-          "annotations": {}
-        },
-        {
-          "comment": null,
-          "name": "checksum",
-          "default": null,
-          "nullok": true,
-          "type": {
-            "typename": "text"
-          },
-          "annotations": {}
-        }
-      ],
-      "annotations": {
-        "tag:isrd.isi.edu,2016:visible-columns" : {
-          "*" : ["id", "uri", "fk_id"]
+        "referee": {
+            "comment": null,
+            "kind": "table",
+            "keys": [
+                {
+                    "comment": null,
+                    "annotations": {},
+                    "unique_columns": [
+                        "id"
+                    ]
+                }
+            ],
+            "table_name": "referee",
+            "schema_name": "upload",
+            "foreign_keys": [],
+            "column_definitions": [
+                {
+                    "comment": null,
+                    "name": "id",
+                    "default": null,
+                    "nullok": false,
+                    "type": {
+                        "typename": "int8"
+                    },
+                    "annotations": {}
+                }
+            ],
+            "annotations" : {}
         }
-      }
     },
-    "referee": {
-      "comment": null,
-      "kind": "table",
-      "keys": [
-        {
-          "comment": null,
-          "annotations": {},
-          "unique_columns": [
-            "id"
-          ]
+    "table_names": [
+        "file",
+        "referee"
+    ],
+    "comment": null,
+    "annotations": {
+        "tag:misd.isi.edu,2015:display": {
```

Comment:
This is something related to the previous schema and not related to your PR. But since you're changing this PR, please remove this unnecessary annotation.